javascript/javascript2/week1/spiritAnimalFinder/spiritAnimalFinder.css

```diff
@@ -1,26 +1,130 @@
+/*This is not part of the homework, I did it for CSS revision. */
+
 *{
     margin: 0;
     padding: 0;
     box-sizing: border-box;
     font-family: 'Times New Roman', Times, serif;
 }
 
+/* Pardon me for choice of color. I will change it if I get time. I running little behind the schedule*/
+
 body {
-   background-color: olivedrab;
+   background-color: rgb(178, 215, 238);
+   display: flex;
    align-items: center;
 }
 
+/*Background-image (gradient pattern) is not fitting properly in the body tag, so I am making it under a container tag. I need to look into this later on*/
```

Comment:
Gradients tend to fit improperly! I will let you do the research on how to avoid the gradient background *repeating* itself. 😉